### PR TITLE
I-care

### LINKED ISSUE #16: I-Care: Add I-Care section
Carousel on right/left + text

Same text and images as last year

### PR DESCRIPTION
**Pull request by tarun-hegde**
### PR description
Added modal to I-CARE
> Cover things that have been added and/or removed

### CODE REVIEW

File: src/components/Icare.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  const buttonText = showMore ? "Show Less" : "Show More";
+
+  return (
+    <section id="icare">
+    <motion.div className=" h-fit lg:rounded-xl shadow-sm flex flex-row justify-center flex-wrap my-5 py-3" 
+     initial={{ x: "-50%", opacity: 0 }}
+      whileInView={{ x: 0, opacity: 1 }}
+      transition={{ duration: 0.75 }}>
+    {/* I-care Content */}
+      <div className="sm:basis-full flex item-center justify-center text-center font-[ARMRegular] text-6xl m-3 p-4 text-primary font-semibold">
```
Comment (mittal-parth) [inline src/components/Icare.jsx]:
```suggestion
      <div className="sm:basis-full flex item-center justify-center text-center font-[ARMRegular] text-6xl text-gradient m-3 p-4 text-primary font-semibold">
```

File: src/components/Icare.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import React,{useState} from 'react'
+import { motion } from "framer-motion";
+import ICareModal from './ICareModal';
+import {stunt} from '../assets/index';
+import { GiCrossedSabres } from "react-icons/gi";
+const Icare = () => {
+  const [showMore, setShowMore] = useState(false);
+  const buttonText = showMore ? "Show Less" : "Show More";
```
Comment (mittal-parth) [inline src/components/Icare.jsx]:
```suggestion
  const buttonText = showMore ? "Close" : "Read More";
```

**Comment by mittal-parth** [comment]
- [x] Use some library for the carousel instead of writing logic for it. Its quite hacky and not smooth. You can look at what is being used for Events.
- [x] Retain the buttons to navigate in the carousel
- [x] The modal needs more padding
- [x] The close button should be on the top right corner (basically take the design from competitions, the padding and everything is proper there)
- [x] Keep the heading left aligned for bigger devices and centered for smaller ones (this is what we are following now)
- [x] There had to be some content on the left side and the image on the right even on default. When you open, you get all the content and the carousel.

**Comment by mittal-parth** [comment]
- Keep the text size same as About Us and same text-justified
- The asset they have given is honesty not nice?
- The carousel still has flashes sometimes, I dont know why

Rest all is fine

**Comment by tarun-hegde** [comment]
> * Keep the text size same as About Us and same text-justified
> * The asset they have given is honesty not nice?
> * The carousel still has flashes sometimes, I dont know why
> 
> Rest all is fine


yeah the asset isn't great

**Comment by mittal-parth** [comment]
How about this - https://lottiefiles.com/74130-education
Change colour scheme obviously 

Or this - https://lottiefiles.com/88020-creative-teaching-education-animation-icons
https://lottiefiles.com/45997-hand-with-love

**Pull request merged**